docs/source/technical_tutorials/visualization/interval_color_mapping.ipynb

```diff
@@ -2,19 +2,24 @@
  "cells": [
```

Comment:
<p>I don't think this tutorial is making use of pystac-client but it certainly could with a few changes!</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/MAAP-Project/maap-documentation/pull/253/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>